source/compatibility.txt

```diff
@@ -0,0 +1,61 @@
+.. _php-compatibility:
+
+=============
+Compatibility
+=============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: backwards compatibility, versions, upgrade
+
+MongoDB Compatibility
+---------------------
+
+The following compatibility table specifies the recommended version or versions
+of the PHP driver for use with a specific version of MongoDB.
+
+The first column lists the driver version.
+
+.. sharedinclude:: dbx/lifecycle-schedule-callout.rst
+
+.. include:: /includes/mongodb-compatibility-table-php.rst
+
+For more information on how to read the compatibility tables, see our guide on
+`MongoDB Compatibility Tables <about-driver-compatibility>`__.
```

Comment:
S: think link is malformed - I think it is meant to link here: https://www.mongodb.com/docs/drivers/about-compatibility/

so you can change to:
```suggestion
For more information on how to read the compatibility tables, see our guide on
:ref:`MongoDB Compatibility Tables <about-driver-compatibility>`.
```